internal/failover/ping_windows.go

```diff
@@ -0,0 +1,19 @@
+package failover
```

Comment:
should we add:

```go
//go:build windows
```

Reply:
No need to. It's Go's convention if the filename ends with `_windows.go` to compile it when `GOOS=windows` only.